rows.go

```diff
@@ -27,7 +27,7 @@ var (
 	maxChunkDownloaderErrorCounter = 5
 )
 
-type snowflakeRows struct {
+type SnowflakeRows struct {
```

Comment:
before I add comments to all of the following, I'd like to confirm this is the one and only way we can pursue for this:

<img width="1354" alt="Screen Shot 2022-01-19 at 1 03 27 AM" src="https://user-images.githubusercontent.com/71045105/150038575-57f20282-8df4-40ee-bc4d-0aec585728b2.png">

Reply:
not applicable anymore with new commit (amended).